### ISSUE TITLE
install-deps needs to support 15.04

### ISSUE THREAD

**Issue by soumith** [torch/distro#35]
issue to track ubuntu 15.04 support

**Comment by PrincipalComponent** [comment]
Does anyone if this is working? I still can't get Torch to run on 15.04. Thanks

**Comment by hughperkins** [comment]
15.10 works fine for me.  At least, torch runs on 15.10 just fine for me, and I installed 15.10 a few times, and it worked each time.  A few observations:
- I needed to install libreadline-dev by hand at least once.  I cant remember if this was because I was too lazy to run the install-deps script, or whether it is missing from the install-deps script
- I had to each time do `sudo apt-get install libblas-dev liblapack-dev` Again, might be because i was too lazy to run `install-deps`,  cant remember
- importantly, because default compiler on 15.10 is gcc 5.x, I had to add the following to `torch-activate` to be able to build against cuda:
```
export CC=gcc-4.9
export CXX=g++-4.9
```

Otherwise, everything 'just works'.

(By the way 15.04 is on 9 months support, so it's no longer supported; but 15.10 is supported, by Canonical, until July this year)
